README.md

```diff
@@ -57,6 +57,32 @@ Now, you can run your application by running ```docker container run --name g5v
 G5V requires G5API to be running at `/api`, you can change this behavior run `docker build -t yourname\g5v:latest -f DockerfileFull --build-arg VUE_APP_G5V_API_URL=<your_api_url>` (be sure to include `http(s)://` and not to have a trailing `/`). If you are using Docker Compose refer to [this](https://docs.docker.com/compose/compose-file/compose-file-v3/#args).
 Some example setup configs can be found [here](https://github.com/PhlexPlexico/G5V/wiki).
 
+#### DockerfileFullCaddy:
```

Comment:
Just make sure the readme doesn't get changed too much either - a rebase should fix these :)